### PR TITLE
feat(topbar/user): add onclick feature in atom button

### PR DESCRIPTION
**Pull request by siscu**
Add onClick feature in atom button

### CODE REVIEW

**Review by davidbarna — APPROVED** [review]

File: components/topbar/user/src/index.js [hunk 1]
```diff
@@ -185,28 +185,28 @@ class TopbarUser extends Component {
     const toggleMenuClassName = cx('sui-TopbarUser-toggle', {
       'has-notifications': hasNotifications
     })
-    const {icon: NavCtaIcon, url: navCtaUrl, text: navCtaText} = navCTA
+    const { icon: NavCtaIcon, url: navCtaUrl, text: navCtaText, onClick: navCtaOnClick = () => ({}) } = navCTA
```
Comment (kikoruiz) [inline components/topbar/user/src/index.js]: You can resume it like this `() => {}`

File: components/topbar/user/src/index.js [hunk 2]
```diff
@@ -246,14 +246,15 @@ class TopbarUser extends Component {
...
             link
             href={navCtaUrl}
             title={navCtaText}
-            leftIcon={<navCTA.icon svgClass="sui-TopbarUser-ctaButtonIcon" />}
-            size="small"
-            type="accent"
+            leftIcon={<navCTA.icon svgClass='sui-TopbarUser-ctaButtonIcon' />}
+            size='small'
+            type='accent'
+            onClick={navCtaOnClick}
```
Comment (davidbarna) [inline components/topbar/user/src/index.js]: The prop name should be more specific IMO.
onCTAClick, onLoginClick, etc

**Comment by kikoruiz** [comment]
Are the style changes in the code properly done? Are them "prettier compliant"?

File: components/topbar/user/src/index.js (hunk 1 above)
Comment (siscu) [reply components/topbar/user/src/index.js]: ok, changed

Comment (davidbarna) [reply components/topbar/user/src/index.js]: `() => ({})` returns an empty object, `() => {}` returns undefined.
Nevermind maybe a defaultProp would be more accurate.

File: components/topbar/user/src/index.js (hunk 2 above)
Comment (siscu) [reply components/topbar/user/src/index.js]: 👍  Thanks, changed prop name.

**Review by kikoruiz — APPROVED** [review]

**Pull request merged**